Example/OriginateUI/GradientViewController.m

```diff
@@ -0,0 +1,177 @@
+//
+//  GradientViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 6/22/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "GradientViewController.h"
+#import "CodeViewController.h"
+#import <QuartzCore/QuartzCore.h>
+@import OriginateUI;
+
+@interface GradientViewController ()
+
+@property UILabel *label;
+@property UILabel *label1;
+@property UILabel *label2;
+@property OriginateGradientView *gradientView;
+@property (nonatomic, strong) OriginateValidatedTextField *textFieldFirst;
+@property (nonatomic, strong) OriginateValidatedTextField *textFieldSecond;
+@property BOOL firstValid;
+@property BOOL secondValid;
+
+@end
+
+@implementation GradientViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
```

Comment:
If we do that we don't need a setter for the properties either. If you make them `readonly` you'll have to synthesize the underlying instance variables though.